libs/designsystem/menu/src/menu.component.scss

```diff
@@ -0,0 +1,28 @@
+@use '../../../../node_modules/@kirbydesign/core/src/scss/interaction-state/index';
+@use '../../../../node_modules/@kirbydesign/core/src/scss/utils';
```

Comment:
We should be able to import these this like so: `@use '@kirbydesign/core/src/scss/utils';`